src/components/common/header/PageHeader.tsx

```diff
@@ -11,6 +10,13 @@ export interface PageHeaderType {
     tippyRedirectLink?: string
     CreateButtonIcon?: React.FunctionComponent<any>
     showIconBeforeText?: boolean
+    showTabs?: boolean
+    renderHeaderTabs?: () => void
```

Comment:
return type of renderHeaderTabs and breadcrumb should be jsx:element right?

Reply:
Both are the callback function, not the jsx element